shoop/admin/templates/shoop/admin/products/_edit_base_form.jinja

```diff
@@ -15,7 +15,11 @@
     {{ bs3.field(product_form.barcode) }}
     {{ bs3.field(product_form.gtin) }}
     {{ bs3.field(product_form.category) }}
-    {{ bs3.field(product_form.stock_behavior) }}
```

Comment:
Maybe -- for this once -- leave the removed code in the comment, to make it clear what there is to "enable" in the near future :)